components/feature/search/src/main/java/mozilla/components/feature/search/middleware/SearchMiddleware.kt

```diff
@@ -41,7 +47,8 @@ class SearchMiddleware(
     private val customStorage: CustomStorage = CustomSearchEngineStorage(context),
     private val bundleStorage: BundleStorage = BundledSearchEnginesStorage(context),
     private val metadataStorage: MetadataStorage = SearchMetadataStorage(context),
-    private val ioDispatcher: CoroutineContext = Dispatchers.IO
+    private val ioDispatcher: CoroutineContext = Dispatchers.IO,
+    private val applicationSearchEngines: List<SearchEngine> = emptyList(),
```

Comment:
Maybe reorder this above ioDispatcher. Typically we just see the IO as the last parameter.

Reply:
As agreed, I've remove `applicationSearchEngines` as a parameter.